src/Aquarius.Client/Samples/Client/ServiceModel.cs

```diff
@@ -2361,7 +2366,7 @@ public class GetSpecimens : IReturn<SearchResultSpecimen>, IPaginatedRequest
         public string Cursor { get; set; }
         public Instant? EndModificationTime { get; set; }
         public List<string> FieldTripIds { get; set; }
-        public string FieldVisitId { get; set; }
+        public List<string> FieldVisitId { get; set; }
```

Comment:
@EricHeath-AI @JesseChezenko-AI This is the breaking change in the generated service model which caught my eye.